Shoko.Server/Server/ShokoServer.cs

```diff
@@ -722,7 +713,9 @@ private void WorkerSetupDB_DoWork(object sender, DoWorkEventArgs e)
 
             if (ServerSettings.Instance.Import.RunOnStart && folders.Count > 0)
             {
-                RunImport();
+                var schedulerFactory = ServiceContainer.GetService<ISchedulerFactory>();
```

Comment:
Use GetRequiredService if you do not want the possibility of null. It will throw it the service is not found instead of returning null and needing a check.